sentry-core/src/main/java/io/sentry/core/Hub.java

```diff
@@ -173,9 +173,8 @@ public void close() {
         }
       } catch (Exception e) {
         logIfNotNull(options.getLogger(), SentryLevel.ERROR, "Error while closing the Hub.", e);
-      } finally {
-        isEnabled = false;
       }
+      isEnabled = false;
```

Comment:
There is a severe performance penalty for using finalizers and this line of code could be done without a `finally`

Reply:
With `finalizers` you mean calling `Object.finalize()`? I don't know of any penalty in using a `finally` block.

Reply:
@ninniuz yes, the `Object.finalize()`, but I could not explain correctly, I mixed up the meaning.
Usually for closing/cleaning is common to have a `finalizer`, but it's bad on Java.
In this case, as we don't have any `return`, the `finally` is just not necessary.